base/meta.jl

```diff
@@ -56,4 +59,22 @@ macro dump(expr)
     dump(expr)
 end
 
+"""
+    leaftypes(t::Type) -> Vector{Union{DataType, UnionAll}}
+
+Return an array of all concrete subtypes of `t`, including `t` if it is concrete.
```

Comment:
Perhaps: 

>Return a vector of all concrete subtypes of `t`.
> If `t` is concrete  then this array contains `t` itself and no other elements.

idk,
I feel line "including" is the wrong word,
since either the Array contains just `t`, or it does not contain `t` at all.
Never both.

Reply:
Ah, you're right. I mirrored the `abstracts` docstring without thinking too hard about it.